install.rst

```diff
@@ -18,49 +18,136 @@
 Installing Zeek
 ===============
 
-Prerequisites
+To run Zeek, grab our official Docker images, download our Linux binary
+packages, install via Homebrew on your Mac, use the ports collection on FreeBSD,
+or build Zeek yourself.
+
+Docker Images
 =============
 
-Before installing Zeek, you'll need to ensure that some dependencies
-are in place.
+We provide `official Docker images <https://hub.docker.com/u/zeekurity>`_ on the
+Docker Hub. These images are based on Debian 11 and feature a complete Zeek
+installation with ``zeek``, ``zkg``, and the Spicy toolchain, but are otherwise
+minimal to avoid bloat in derived images. For example, if you'd like to install
+Zeek plugins in those images, you'll need to install the needed compiler
+toolchain.
 
-Required Dependencies
----------------------
+We provide images for multiple releases:
 
-Zeek requires the following libraries and tools to be installed
-before you begin:
+    * For the latest feature release: ``docker pull zeekurity/zeek:latest``
+    * For the latest LTS release: ``docker pull zeekurity/zeek:lts``
+    * For a specific release: ``docker pull zeekurity/zeek:5.0.0-rc1``
+    * For the nightly build: ``docker pull zeekurity/zeek-dev:latest``
 
-    * Libpcap                           (http://www.tcpdump.org)
-    * OpenSSL libraries                 (https://www.openssl.org)
-    * BIND8 library
-    * Libz
-    * Bash (for ZeekControl)
-    * Python 3.5 or greater             (https://www.python.org/)
+The Dockerfile lives `here <https://github.com/zeek/zeek/blob/master/docker/Dockerfile>`_.
 
-To build Zeek from source, the following additional dependencies are required:
+Binary Packages
+===============
 
-    * CMake 3.15 or greater              (https://www.cmake.org)
-    * Make
+Linux
+-----
+
+We provide `binary packages <https://build.opensuse.org/project/show/security:zeek>`_
+for a wide range of Linux distributions via the `openSUSE Build Service
+<https://build.opensuse.org/>`_. To install, first add the relevant OBS
+package repository to your system, then use your system's package manager
+as usual. Our packages resolve third-party dependencies for you.
```

Comment:
```suggestion
as usual.
```
Might be opinionated, but seems the contrary would be worth mentioning rather than the expected behavior.

Reply:
Done